### ISSUE TITLE
error displaying legend in chrome

### ISSUE THREAD

**Issue by ignacio82** [rstudio/leaflet#129]
This is my code:

    ---
    title: "Map"
    output: html_document
    runtime: shiny
    ---
    
    ## Test
    
    ```{r texas, message=F, echo=F}
    library(dplyr)
    library(sp)
    library(leaflet)
    circleFun <- function(center = c(0,0),diameter = 1, npoints = 100){
      r = diameter / 2
      tt <- seq(0,2*pi,length.out = npoints)
      xx <- center[1] + r * cos(tt)
      yy <- center[2] + r * sin(tt)
      Sr1 = Polygon(cbind(xx, yy))
      Srs1 = Polygons(list(Sr1), "s1")
      SpP = SpatialPolygons(list(Srs1), 1:1)
      return(SpP)
    }
    Circle.Town <- circleFun(c(1,-1),2.3,npoints = 100)
    
    df1 <- data.frame(long=c(0.6,1,1.4), lat=c(-2, -.8, -0.2), other=c('a', 'b', 'c'), X=c(10,8,6))
    
    # Create a continuous palette function
    pal <- colorNumeric(
      palette = "RdYlBu",
      domain = df1$X
    )
    
    leaflet(height = "600px", width = "800px") %>% 
      addTiles() %>%
      addPolygons(data = Circle.Town, color = 'green',  fillOpacity = .7) %>%
      addCircleMarkers(data = df1, lat = ~lat, lng =~long, 
                       radius = ~X, 
                       fillColor = ~pal(X),
                       stroke = FALSE, fillOpacity = 1,
                       clusterOptions = markerClusterOptions()) %>% 
      addLegend(position = "topright",
              pal = pal, values = df1$X,
              title = "X",
              opacity = 1
              ) %>% 
      setView(lng = 1, lat = -1, zoom = 8)
    
    ```
    
The numbers on the legend are not displayed correctly in chrome:

![Imgur](http://i.imgur.com/PftXBQS.png)

But if I open the html in ie, it looks fine.

![Imgur](http://i.imgur.com/0Rqgp62.png)

Thanks for the help!

**Comment by yihui** [comment]
Thanks for the reproducible example. I'll investigate the issue soon. (Wow, IE works but Chrome doesn't :)

**Comment by yihui** [comment]
Fixed via #131.
